### PR TITLE
added styling for card

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added styling for card component.

### CODE REVIEW

File: css/components/card/index.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  width:fit-content;
+  justify-content: flex-start;
+  cursor: pointer;
+  /* gap:1; */
+  /* align-items: center; */
+background-color: var(--light-color);
+    padding: 0.5rem;
+    /* box-shadow: 0px 0px 0.5rem rgba(0, 0, 0, 0.2); */
+    box-shadow: var(--bx-sh-2);
+    border-radius: var(--br-radius);
```
Comment (pankaj-prog) [inline css/components/card/index.css]: removed commented styles

File: css/components/card/index.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  }
+  .card-text-overlay
+  {position: absolute;
+    top:.5rem;left:.5rem;
+    font-size: .9rem;
+    color: var(--light-color);
+    background-color: var(--primary-color);
+    padding:.3rem .6rem;
+    border-radius: .3rem;
+  }
```
Comment (pankaj-prog) [inline css/components/card/index.css]: improve the indentation

File: css/components/card/index.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+   
+      font-size: .8rem;
+      display:flex;
+      flex-flow: row wrap;
+      row-gap: .5rem;
+      column-gap: .5rem;
+      justify-content: space-between;
+        padding: 0.5rem;
+        border-radius: 0 0 0.125rem 0.125rem;
+    }
```
Comment (pankaj-prog) [inline css/components/card/index.css]: you can use px for such small units

**Review by pankaj-prog — CHANGES_REQUESTED** [review]

File: css/components/card/index.css (hunk 3 above)
Comment (sanjitsarkar) [reply css/components/card/index.css]: Okay I will fix that.

File: css/components/avatar/index.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+.avatar-badge-container{
+position: relative;
+/* transform:scale(1);
+    border-radius: 50%;
+    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    box-shadow: var(--bx-sh-3); */
+}
+.avatar-badge{
+   right: 0%;
+    top:0%;
```
Comment (PrashanthreddyAinala) [inline css/components/avatar/index.css]: Just 0 will also fine no need of %.

File: css/components/button/index.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+}
+.btn:active{
+    box-shadow: var(--bx-sh-3);
+    opacity:.7;
+
+    transform: scale(.95);
+}
+  .btn-disabled {
+    background-color: #dddbdb;
+
```
Comment (PrashanthreddyAinala) [inline css/components/button/index.css]: use variable colors

File: css/reset.css [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  box-sizing: border-box;
+}
+body {
+  font-family: Poppins;
+}
+
+hr {
+  border: 0;
+  border-top: 1px dotted #efefef;
+}
```
Comment (PrashanthreddyAinala) [inline css/reset.css]: same here for color

File: css/util/color/index.css [hunk 7]
```diff
@@ -0,0 +1,41 @@
+
+.bg-primary {
+    background-color: var(--primary-color)  !important;
+}
```
Comment (PrashanthreddyAinala) [inline css/util/color/index.css]: In this complete file avoid using important, that should be the last preference.

**Review by PrashanthreddyAinala — COMMENTED** [review]
Rest all the code looks great

**Review by pankaj-prog — APPROVED** [review]

File: css/components/card/index.css [hunk 8]
```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  cursor: pointer;
+  background-color: var(--light-color);
+  padding: 0.5rem;
```
Comment (deekshithmd) [inline css/components/card/index.css:7]: use px for small value

File: css/components/card/index.css [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.card-footer {
+  display: flex;
+  flex-flow: row wrap;
+  row-gap: 0.5rem;
+  column-gap: 0.5rem;
+  justify-content: space-between;
+  padding: 0.5rem;
+  border-radius: 0 0 0.125rem 0.125rem;
```
Comment (deekshithmd) [inline css/components/card/index.css:96]: for border radius use px

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**